docs/generators/python.md

```diff
@@ -67,6 +67,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 <ul class="column-ul">
 <li>all_params</li>
 <li>and</li>
+<li>args</li>
```

Comment:
Should arg also be added?
What do you think about instead converting all arg into _arg and args into _args
That way they can stay non-reserved?

Reply:
Do you mean do these conversions in the templates?

Reply:
Yes in the templates